src/coreclr/vm/win32threadpool.cpp

```diff
@@ -505,2987 +342,464 @@ void ThreadpoolMgr::InitPlatformVariables()
         CONTRACT_VIOLATION(GCViolation|FaultViolation);
         hNtDll = CLRLoadLibrary(W("ntdll.dll"));
         _ASSERTE(hNtDll);
-        if (!UsePortableThreadPool())
-        {
-            hCoreSynch = CLRLoadLibrary(W("api-ms-win-core-synch-l1-1-0.dll"));
-            _ASSERTE(hCoreSynch);
-        }
     }
 
     // These APIs must be accessed via dynamic binding since they may be removed in future
     // OS versions.
     g_pufnNtQueryInformationThread = (NtQueryInformationThreadProc)GetProcAddress(hNtDll,"NtQueryInformationThread");
     g_pufnNtQuerySystemInformation = (NtQuerySystemInformationProc)GetProcAddress(hNtDll,"NtQuerySystemInformation");
```

Comment:
`g_pufnNtQueryInformationThread` and `g_pufnNtQuerySystemInformation` are only used in portable threadpool code. You can delete them and everything that calls them.